src/lib/router.ts

```diff
@@ -241,6 +241,20 @@ function pathToRoute(path: string): Route | null {
       }
       return null;
     }
+    case "session": {
+      const id = segments.shift();
+      if (id) {
+        return {
+          resource: "session",
+          params: {
+            id,
+            sig: url.searchParams.get("sig") ?? "",
+            pk: url.searchParams.get("pk") ?? "",
```

Comment:
Is `pk` a public key? If so we should probably use `publicKey` as the variable and query param.

Reply:
Hey yeah I shortened it to avoid such a long querystring, but since the user won't type it by hand.
I have no issue in writing it out.

Reply:
Ok this should be fixed with 3d9e644aaf039c6630e8b0cd54c052708746678b

Reply:
I'm keeping the serialized querystring with pk and sig, since `radicle-httpd` is expecting that, we can eventually change that, but the PR on the heartwood repo just got merged.